### PR TITLE
Cleanup apache tutorials

### PR DESCRIPTION
**Pull request by squidboylan**
Make some small changes to apache tutorials in compliance with the new styleguide

### CODE REVIEW

File: source/dreamcompute/tutorials/configure-apache-on-fedora-or-centos.rst [hunk 1]
```diff
@@ -2,27 +2,28 @@
...
+    These instructions assume you run a Fedora- or CentOS-based system as
+    they have their own specific configuration and file hierarchy.
 
 Installing Apache
 ~~~~~~~~~~~~~~~~~
 
 To install Apache on your system, run the following command:
 
-.. code::
+.. code-block:: bash
```
Comment (smaffulli) [inline source/dreamcompute/tutorials/configure-apache-on-fedora-or-centos.rst]: why did you change this? do we need to change the syntax on other .. code:: directives? (like the lines below, for example?)

**Comment by squidboylan** [comment]
I just changed it to be consistent with the other one, it's not a syntax error. Not sure which we want to use

**Comment by justinknierim** [comment]
lgtm btw
